docs/core_docs/docs/troubleshooting/errors/INVALID_GRAPH_UPDATE.mdx

```diff
@@ -0,0 +1,40 @@
+# INVALID_GRAPH_UPDATE
+
+A LangGraph app received an unexpected return value from a node and failed to update its state at the end of a step.
```

Comment:
are we certain this is the only condition that creates this? can it also be an input with a thread?

Reply:
Hmm probably not, might be worth splitting the error more granularly

Reply:
Yeah let's actually split this into different cases and keep this focused on parallel updates